.github/workflows/pytest_otel.yml

```
@@ -5,10 +5,10 @@ on:
   workflow_dispatch:
   pull_request:
     paths:
-      - '/resources/scripts/pytest_otel/**'
+      - 'resources/scripts/pytest_otel/**'
```

Can you also add this workflow too?

```suggestion
      - 'resources/scripts/pytest_otel/**'
      - '.github/workflows/pytest_otel.yml'
```